scripts/generate-fixture-snapshots.sh

```
@@ -0,0 +1,413 @@
+if [ -n "$RESTRICT_SOURCE_RC" ]; then
+    # Filter to the named tag.
+    if ! printf '%s\n' "$ALL_TAGS" | grep -Fxq "$RESTRICT_SOURCE_RC"; then
+        echo "generator: --source-rc '$RESTRICT_SOURCE_RC' not in eligible set" >&2
+        echo "  eligible: $(printf '%s' "$ALL_TAGS" | tr '\n' ' ')" >&2
+        exit 2
+    fi
+    ALL_TAGS="$RESTRICT_SOURCE_RC"
+fi
+
+# Build the (source-rc, variant) pair list.
+pairs=""
```

Shellcheck (0.11.0)

[warning] pairs appears unused. Verify use (or export if used externally).

(SC2034)